server/routes/employeeRoutes.js

```diff
@@ -5,9 +5,10 @@ import AuthValidator from '../middlewares/authValidator';
 import asyncErrorHandler from '../helpers/asyncErrorHandler';
 import isLogged from '../middlewares/isLogged';
 import isAdmin from '../middlewares/isAdmin';
+import retrieve from '../models/retrieveEmployees';
 
 const router = Router();
 
 router.post('/employee', isLogged, isAdmin, AuthValidator.addEmployeeValidator, asyncErrorHandler(EmployeeController.addEmployee));
-
+router.get('/retrieve/employees', isLogged, isAdmin, retrieve);
```

Comment:
You can also wrap this controller into the `asyncErrorHandler` function imported above, just to handle error that might occur in your controller too.